testplan/web_ui/testing/package.json

```diff
@@ -32,6 +32,7 @@
     "pegjs": "^0.11.0-master.b7b87ea",
     "plotly.js": "^1.58.5",
     "react": "16.13.1",
+    "react-app-polyfill": "1.0.6",
```

Comment:
It appears that NPM/Yarn flattens the hierarchy, but PNPM uses links in the background and every explicit import needs to be present in dependencies.